CONTRIBUTING.md

```diff
@@ -62,7 +62,7 @@ If you are proposing a feature:
 ## Documentation
 
 The latest API documentation is usually available
-[here](https://airflow.incubator.apache.org/). To generate a local version,
+[here](https://airflow.apache.org/). To generate a local version,
```

Comment:
Can we define what we mean by latest? Latest release, or master? (We make this distinction in README.md, here to would be nice)